sortfilterproxymodel.qbs

```diff
@@ -59,6 +61,7 @@ Product
         [
             "filter.cpp",
             "qqmlsortfilterproxymodel.cpp",
+            "qqmlsortfilterproxymodel_plugin.cpp",
```

Comment:
Ah should we? that file was automatically created by QtCreator when I selected the type QML Plugin.